api/api-service.js

```diff
@@ -9,20 +9,19 @@ var Seneca = require('seneca')
 
 var server = new Hapi.Server()
 
-server.connection({ 
+server.connection({
   port: PORT
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
-      legacy: { logging: false },
-      internal: { logger: require('seneca-demo-logger') },
       debug: {short_logs:true}
     })
+    .use('demo-logger')
```

Comment:
OK - I see what's happening - the default logger is acting on the log:silent setting
This needs to be removed.

The `internal: {logger:` is not a hack - logging needs to be in place from the statt of execution, so needs a special way to initialize - normal plugin loading is too late.

@mcdonnelldean no it isn't :) Seneca core does not do log filtering - all does it dump JSON output onto the logging plugins - they do their own filtering